### PR TITLE
DOC-2283: replace RedisJSON with JSON

### PR DESCRIPTION
**Pull request by dwdougherty**
This commit is for the module de-branding effort.

### CODE REVIEW

File: docs/docs/indexing_JSON.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 title: Index/Search JSON documents
 linkTitle: Index/Search
 weight: 2
-description: Combine RedisJSON and RediSearch to index and search JSON documents
+description: Combine Redis JSON and Search and Query to index and search JSON documents
 ---
 
-In addition to storing JSON documents, you can also index them using the [RediSearch](/docs/stack/search) module. This enables full-text search capabilities and document retrieval based on their content.
+In addition to storing JSON documents, you can also index them using the [Search and Query](/docs/stack/search) module. This enables full-text search capabilities and document retrieval based on their content.
```
Comment (elena-kolevska) [inline docs/docs/indexing_JSON.md:8]:
```suggestion
In addition to storing JSON documents, you can also index them using the [Search and Query](/docs/stack/search) feature. This enables full-text search capabilities and document retrieval based on their content.
```

File: docs/docs/_index.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ type: docs
 [![Discord](https://img.shields.io/discord/697882427875393627?style=flat-square)](https://discord.gg/QUkjSsk)
 [![Github](https://img.shields.io/static/v1?label=&message=repository&color=5961FF&logo=github)](https://github.com/RedisJSON/RedisJSON/)
 
-The RedisJSON module provides JSON support for Redis. RedisJSON lets you store, update, and retrieve JSON values in a Redis database, similar to any other Redis data type. RedisJSON also works seamlessly with [RediSearch](https://redis.io/docs/stack/search/) to let you [index and query JSON documents](https://redis.io/docs/stack/search/indexing_json).
+The Redis JSON module provides JavaScript Object Notation support for Redis. Redis JSON lets you store, update, and retrieve JSON values in a Redis database, similar to any other Redis data type. Redis JSON also works seamlessly with [Search and Query](https://redis.io/docs/stack/search/) to let you [index and query JSON documents](https://redis.io/docs/stack/search/indexing_json).
```
Comment (elena-kolevska) [inline docs/docs/_index.md:11]: @adrianoamaral  can you please reword this? 🙏

Comment (elena-kolevska) [inline docs/docs/_index.md:11]:
```suggestion
Redis JSON provides JavaScript Object Notation (JSON) support for Redis. It lets you store, update, and retrieve JSON values in a Redis database, similar to any other Redis data type. Redis JSON also works seamlessly with [Search and Query](https://redis.io/docs/stack/search/) to let you [index and query JSON documents](https://redis.io/docs/stack/search/indexing_json).
```

**Comment by dwdougherty** [comment]
Changes were covered by other commits.

**Pull request closed without merging**